WindowsServerDocs/failover-clustering/deploy-two-node-clustered-file-server.md

```diff
@@ -311,7 +311,9 @@ To configure a file server failover cluster, follow the below steps.
 
    ![File Server Type](media/Cluster-File-Server/Cluster-FS-File-Server-Type.png)
 
-8. In the **Client Access Point** window, input the name of the file server you will be using.  Please note that this is not the name of the cluster.  This is for the file share connectivity.  For example, if I want to connect to \\SERVER, the name inputted would be SERVER.
+8. In the **Client Access Point** window, input the name of the file server you will be using.  Please note that this is not the name of the cluster.  
+This is for the file share connectivity.  
+For example, if you want to connect to \\\\SERVER, specify the name as SERVER.
 
    > [!NOTE]
    > If you are using static IP Addresses, you will need to select the network to use and input the IP Address it will use for the cluster name.  If you are using DHCP for your IP Addresses, the IP Address will be configured automatically for you.
```

Comment:
Thank you for the positive reply. May I ask you to make these ones lowercase too? (or you can merge my suggestion, if you prefer)
```suggestion
   > If you are using static IP addresses, you will need to select the network to use and input the IP address it will use for the cluster name.  If you are using DHCP for your IP addresses, the IP address will be configured automatically for you.
```